### PR TITLE
[gui-tests][full-ci] Cleanup account from UI after each test

### PR DESCRIPTION
**Pull request by saw-jan**
In Windows (specially when running tests against oc10 and ocis simultaneously), removing sync folders and config file is not enough for cleanup so this PR add the accounts cleanup from the UI level as well. Removing accounts from client UI removes the sync folder entries from the Windows registry which helps in proper cleanup.

### Related Issues:
Part of https://github.com/owncloud/client/issues/9833

### CODE REVIEW

File: test/gui/shared/scripts/bdd_hooks.py [hunk 1]
```diff
@@ -216,3 +200,58 @@ def hook(context):
...
+        # In Windows, removing only config and sync folders won't help
+        # so to work around that, remove the account connection
+        close_open_dialogs()
+        server_host = urlparse(get_config('localBackendUrl')).netloc
+        accounts = Toolbar.get_accounts()
+        for account in accounts:
+            displayname = account.split('\n')[0]
+            Toolbar.openAccount(displayname, server_host)
+            AccountSetting.removeAccountConnection()
+        if len(accounts):
```
Comment (TheOneRing) [inline test/gui/shared/scripts/bdd_hooks.py:219]:
```suggestion
        if accounts:
```

File: test/gui/shared/scripts/bdd_hooks.py [hunk 2]
```diff
@@ -216,3 +200,58 @@ def hook(context):
...
+    # Detach (i.e. potentially terminate) all AUTs at the end of a scenario
+    for ctx in applicationContextList():
+        # get pid before detaching
+        pid = ctx.pid
+        ctx.detach()
+        wait_until_app_killed(pid)
+
+    # clean up config files
+    for config_file in os.listdir(get_config('clientConfigDir')):
+        os.unlink(os.path.join(get_config('clientConfigDir'), config_file))
```
Comment (TheOneRing) [inline test/gui/shared/scripts/bdd_hooks.py:231]: Tip: https://docs.python.org/3/library/pathlib.html

File: test/gui/shared/scripts/bdd_hooks.py [hunk 3]
```diff
@@ -216,3 +200,58 @@ def hook(context):
...
+
+    # delete local files/folders
+    for filename in os.listdir(get_config('clientRootSyncPath')):
```
Comment (TheOneRing) [inline test/gui/shared/scripts/bdd_hooks.py:234]: Tip: https://docs.python.org/3/library/os.html#os.scandir

File: test/gui/shared/scripts/bdd_hooks.py [hunk 4]
```diff
@@ -216,3 +200,56 @@ def hook(context):
 
     previousFailResultCount = test.resultCount("fails")
     previousErrorResultCount = test.resultCount("errors")
+
+
+def teardown_client():
+    # Cleanup user accounts from UI for Windows platform
+    # It is not needed for Linux so skipping it in order to save CI time
+    if not isWindows():
```
Comment (TheOneRing) [inline test/gui/shared/scripts/bdd_hooks.py:208]: ?

Comment (saw-jan) [reply test/gui/shared/scripts/bdd_hooks.py:208]: Right now, I am just trying to get the intel on by how much this will increase the CI time

**Comment by saw-jan** [comment]
@TheOneRing what might be these crashes? Maybe because the test is fast for the client? Most of the crashes are related to sync

```c
[Thread debugging using libthread_db enabled]
Using host libthread_db library "/lib64/libthread_db.so.1".
Core was generated by `/drone/src/build/bin/owncloud -s --logfile - --logdebug --logflush'.
Program terminated with signal SIGSEGV, Segmentation fault.
#0  OCC::Folder::syncPaused (this=0x0) at /drone/src/src/gui/folder.cpp:394
394	    return _definition.paused;
[Current thread is 1 (Thread 0x7faaa1e0af00 (LWP 799))]
#0  OCC::Folder::syncPaused() const (this=0x0) at /drone/src/src/gui/folder.cpp:394
#1  0x00000000004e0b26 in OCC::Folder::canSync() const (this=0x0) at /drone/src/src/gui/folder.cpp:399
#2  0x00000000005f8989 in OCC::SyncScheduler::startNext()::$_0::operator()(OCC::SyncResult const&) const::{lambda()#1}::operator()() const::{lambda()#1}::operator()() const (this=0x2938be0) at /drone/src/src/gui/scheduling/syncscheduler.cpp:162
#3  0x00000000005f8916 in QtPrivate::FunctorCall<QtPrivate::IndexesList<>, QtPrivate::List<>, void, OCC::SyncScheduler::startNext()::$_0::operator()(OCC::SyncResult const&) const::{lambda()#1}::operator()() const::{lambda()#1}>::call(OCC::SyncScheduler::startNext()::$_0::operator()(OCC::SyncResult const&) const::{lambda()#1}::operator()() const::{lambda()#1}&, void**) (f=..., arg=0x7ffef77d4f80) at /usr/include/qt6/QtCore/qobjectdefs_impl.h:127
#4  0x00000000005f88e1 in QtPrivate::Functor<OCC::SyncScheduler::startNext()::$_0::operator()(OCC::SyncResult const&) const::{lambda()#1}::operator()() const::{lambda()#1}, 0>::call<QtPrivate::List<>, void>(OCC::SyncScheduler::startNext()::$_0::operator()(OCC::SyncResult const&) const::{lambda()#1}::operator()() const::{lambda()#1}&, void*, void**) (f=..., arg=0x7ffef77d4f80) at /usr/include/qt6/QtCore/qobjectdefs_impl.h:241
#5  0x00000000005f8871 in QtPrivate::QFunctorSlotObject<OCC::SyncScheduler::startNext()::$_0::operator()(OCC::SyncResult const&) const::{lambda()#1}::operator()() const::{lambda()#1}, 0, QtPrivate::List<>, void>::impl(int, QtPrivate::QSlotObjectBase*, QObject*, void**, bool*) (which=1, this_=0x2938bd0, r=0x1ca2b40, a=0x7ffef77d4f80, ret=0x0) at /usr/include/qt6/QtCore/qobjectdefs_impl.h:408
#6  0x00007faaa5b7e80a in QSingleShotTimer::timerEvent(QTimerEvent*) () at /lib64/libQt6Core.so.6
#7  0x00007faaa5b5f6df in QObject::event(QEvent*) () at /lib64/libQt6Core.so.6
#8  0x00007faaa6e4a698 in QApplicationPrivate::notify_helper(QObject*, QEvent*) () at /lib64/libQt6Widgets.so.6
#9  0x00007faaa5b0c018 in QCoreApplication::notifyInternal2(QObject*, QEvent*) () at /lib64/libQt6Core.so.6
#10 0x00007faaa5cb9763 in QTimerInfoList::activateTimers() () at /lib64/libQt6Core.so.6
#11 0x00007faaa5ddc159 in timerSourceDispatch(_GSource*, int (*)(void*), void*) () at /lib64/libQt6Core.so.6
#12 0x00007faaa506e48c in g_main_context_dispatch () at /lib64/libglib-2.0.so.0
#13 0x00007faaa50cc648 in g_main_context_iterate.isra () at /lib64/libglib-2.0.so.0
#14 0x00007faaa506bb13 in g_main_context_iteration () at /lib64/libglib-2.0.so.0
#15 0x00007faaa5ddc45f in QEventDispatcherGlib::processEvents(QFlags<QEventLoop::ProcessEventsFlag>) () at /lib64/libQt6Core.so.6
#16 0x00007faaa5b0c135 in QCoreApplication::processEvents(QFlags<QEventLoop::ProcessEventsFlag>, int) () at /lib64/libQt6Core.so.6
#17 0x00007faa9b89685b in Squish::Hook::notifyApplicationExit() () at /home/headless/squish/lib/libsquishhook.so
#18 0x00007faaa0787a89 in write_event_queue () at /home/headless/squish/lib/libsquishqtwrapper.so
#19 0x00007faaa078b5de in Squish::QtWrapper::Application::exitAUT(int) () at /home/headless/squish/lib/libsquishqtwrapper.so
#20 0x00007faa9b8891fc in Squish::Hook::kill() () at /home/headless/squish/lib/libsquishhook.so
#21 0x00007faa9b897166 in Squish::Hook::handleSend(unsigned int, QString const&, Squish::ArgumentList const&) () at /home/headless/squish/lib/libsquishhook.so
#22 0x00007faa9b94be74 in Squish::AsyncIPC::IncomingSend::finalize(QDataStream&) () at /home/headless/squish/lib/libsquishhook.so
#23 0x00007faa9b949561 in Squish::AsyncIPC::onMessageArrived(QByteArray) () at /home/headless/squish/lib/libsquishhook.so
#24 0x00007faa9b965b41 in Squish::AsyncIPC::qt_static_metacall(QObject*, QMetaObject::Call, int, void**) () at /home/headless/squish/lib/libsquishhook.so
#25 0x00007faaa5b6ddf8 in void doActivate<false>(QObject*, int, void**) () at /lib64/libQt6Core.so.6
#26 0x00007faa9b9663b2 in Squish::AsyncMessageChannel::messageArrived(QByteArray) () at /home/headless/squish/lib/libsquishhook.so
#27 0x00007faa9b950f63 in Squish::TCPMessageChannel::processSingleMessage(QByteArray*) () at /home/headless/squish/lib/libsquishhook.so
#28 0x00007faa9b9525a1 in Squish::TCPMessageChannel::dispatchMessages(bool) () at /home/headless/squish/lib/libsquishhook.so
#29 0x00007faaa5b6ddf8 in void doActivate<false>(QObject*, int, void**) () at /lib64/libQt6Core.so.6
#30 0x00007faaa5b7cbc5 in QSocketNotifier::activated(int, QSocketNotifier::QPrivateSignal) () at /lib64/libQt6Core.so.6
#31 0x00007faaa5b7d3a5 in QSocketNotifier::event(QEvent*) () at /lib64/libQt6Core.so.6
#32 0x00007faaa6e4a698 in QApplicationPrivate::notify_helper(QObject*, QEvent*) () at /lib64/libQt6Widgets.so.6
#33 0x00007faaa5b0c018 in QCoreApplication::notifyInternal2(QObject*, QEvent*) () at /lib64/libQt6Core.so.6
#34 0x00007faaa5ddd47c in socketNotifierSourceDispatch(_GSource*, int (*)(void*), void*) () at /lib64/libQt6Core.so.6
#35 0x00007faaa506e48c in g_main_context_dispatch () at /lib64/libglib-2.0.so.0
#36 0x00007faaa50cc648 in g_main_context_iterate.isra () at /lib64/libglib-2.0.so.0
#37 0x00007faaa506bb13 in g_main_context_iteration () at /lib64/libglib-2.0.so.0
#38 0x00007faaa5ddc45f in QEventDispatcherGlib::processEvents(QFlags<QEventLoop::ProcessEventsFlag>) () at /lib64/libQt6Core.so.6
#39 0x00007faaa5b1912b in QEventLoop::exec(QFlags<QEventLoop::ProcessEventsFlag>) () at /lib64/libQt6Core.so.6
#40 0x00007faaa5b148fd in QCoreApplication::exec() () at /lib64/libQt6Core.so.6
#41 0x0000000000451631 in main(int, char**) (argc=6, argv=0x7ffef77d65e8) at /drone/src/src/gui/main.cpp:449
        resource = {<No data fields>}
        platform = std::unique_ptr<OCC::Platform> = {get() = 0x1b7bb00}
        app = <incomplete type>
        options = {show = true, quitInstance = false, logDir = {d = {d = 0x0, ptr = 0x0, size = 0}, static _empty = 0 u'\000'}, logFile = {d = {d = 0x1c9cf00, ptr = 0x1c9cf10 u"-", size = 1}, static _empty = 0 u'\000'}, logFlush = true, logDebug = true, debugMode = false, userEnforcedLanguage = {d = {d = 0x0, ptr = 0x0, size = 0}, static _empty = 0 u'\000'}, fileToOpen = {d = {d = 0x0, ptr = 0x0, size = 0}, static _empty = 0 u'\000'}}
        singleApplication = {<QObject> = {<No data fields>}, static staticMetaObject = {d = {superdata = {direct = 0x7faaa5fd0460 <QObject::staticMetaObject>}, stringdata = 0x7faaa7aa6240 <(anonymous namespace)::qt_meta_stringdata_KDSingleApplication>, data = 0x7faaa7aa6160 <qt_meta_data_KDSingleApplication>, static_metacall = 0x7faaa7aa1cc0 <KDSingleApplication::qt_static_metacall(QObject*, QMetaObject::Call, int, void**)>, relatedMetaObjects = 0x0, metaTypes = 0x7faaa7aa8b20 <qt_incomplete_metaTypeArray<(anonymous namespace)::qt_meta_stringdata_KDSingleApplication_t, QtPrivate::TypeAndForceComplete<QString, std::integral_constant<bool, true> >, QtPrivate::TypeAndForceComplete<bool, std::integral_constant<bool, true> >, QtPrivate::TypeAndForceComplete<KDSingleApplication, std::integral_constant<bool, true> >, QtPrivate::TypeAndForceComplete<void, std::integral_constant<bool, false> >, QtPrivate::TypeAndForceComplete<QByteArray const&, std::integral_constant<bool, false> >, QtPrivate::TypeAndForceComplete<bool, std::integral_constant<bool, false> >, QtPrivate::TypeAndForceComplete<QByteArray const&, std::integral_constant<bool, false> >, QtPrivate::TypeAndForceComplete<bool, std::integral_constant<bool, false> >, QtPrivate::TypeAndForceComplete<QByteArray const&, std::integral_constant<bool, false> >, QtPrivate::TypeAndForceComplete<int, std::integral_constant<bool, false> > >>, extradata = 0x0}}, d_ptr = std::unique_ptr<KDSingleApplicationPrivate> = {get() = 0x1c9f1e0}}
        folderManager = std::unique_ptr<OCC::FolderMan> = {get() = 0x1ca2370}
        ocApp = std::unique_ptr<OCC::Application> = {get() = 0x1c85720}

<<<<< STACKTRACE END >>>>>
```

**Comment by TheOneRing** [comment]
> @TheOneRing what might be these crashes? Maybe because the test is fast for the client? Most of the crashes are related to sync and they are during the cleanup process (at the end of the tests)
> 
> ```c
> [Thread debugging using libthread_db enabled]
> Using host libthread_db library "/lib64/libthread_db.so.1".
> Core was generated by `/drone/src/build/bin/owncloud -s --logfile - --logdebug --logflush'.
> Program terminated with signal SIGSEGV, Segmentation fault.
> #0  OCC::Folder::syncPaused (this=0x0) at /drone/src/src/gui/folder.cpp:394
> 394	    return _definition.paused;
> [Current thread is 1 (Thread 0x7faaa1e0af00 (LWP 799))]
> #0  OCC::Folder::syncPaused() const (this=0x0) at /drone/src/src/gui/folder.cpp:394
> #1  0x00000000004e0b26 in OCC::Folder::canSync() const (this=0x0) at /drone/src/src/gui/folder.cpp:399
> #2  0x00000000005f8989 in OCC::SyncScheduler::startNext()::$_0::operator()(OCC::SyncResult const&) const::{lambda()#1}::operator()() const::{lambda()#1}::operator()() const (this=0x2938be0) at /drone/src/src/gui/scheduling/syncscheduler.cpp:162
> #3  0x00000000005f8916 in QtPrivate::FunctorCall<QtPrivate::IndexesList<>, QtPrivate::List<>, void, OCC::SyncScheduler::startNext()::$_0::operator()(OCC::SyncResult const&) const::{lambda()#1}::operator()() const::{lambda()#1}>::call(OCC::SyncScheduler::startNext()::$_0::operator()(OCC::SyncResult const&) const::{lambda()#1}::operator()() const::{lambda()#1}&, void**) (f=..., arg=0x7ffef77d4f80) at /usr/include/qt6/QtCore/qobjectdefs_impl.h:127
> #4  0x00000000005f88e1 in QtPrivate::Functor<OCC::SyncScheduler::startNext()::$_0::operator()(OCC::SyncResult const&) const::{lambda()#1}::operator()() const::{lambda()#1}, 0>::call<QtPrivate::List<>, void>(OCC::SyncScheduler::startNext()::$_0::operator()(OCC::SyncResult const&) const::{lambda()#1}::operator()() const::{lambda()#1}&, void*, void**) (f=..., arg=0x7ffef77d4f80) at /usr/include/qt6/QtCore/qobjectdefs_impl.h:241
> #5  0x00000000005f8871 in QtPrivate::QFunctorSlotObject<OCC::SyncScheduler::startNext()::$_0::operator()(OCC::SyncResult const&) const::{lambda()#1}::operator()() const::{lambda()#1}, 0, QtPrivate::List<>, void>::impl(int, QtPrivate::QSlotObjectBase*, QObject*, void**, bool*) (which=1, this_=0x2938bd0, r=0x1ca2b40, a=0x7ffef77d4f80, ret=0x0) at /usr/include/qt6/QtCore/qobjectdefs_impl.h:408
> #6  0x00007faaa5b7e80a in QSingleShotTimer::timerEvent(QTimerEvent*) () at /lib64/libQt6Core.so.6
> #7  0x00007faaa5b5f6df in QObject::event(QEvent*) () at /lib64/libQt6Core.so.6
> #8  0x00007faaa6e4a698 in QApplicationPrivate::notify_helper(QObject*, QEvent*) () at /lib64/libQt6Widgets.so.6
> #9  0x00007faaa5b0c018 in QCoreApplication::notifyInternal2(QObject*, QEvent*) () at /lib64/libQt6Core.so.6
> #10 0x00007faaa5cb9763 in QTimerInfoList::activateTimers() () at /lib64/libQt6Core.so.6
> #11 0x00007faaa5ddc159 in timerSourceDispatch(_GSource*, int (*)(void*), void*) () at /lib64/libQt6Core.so.6
> #12 0x00007faaa506e48c in g_main_context_dispatch () at /lib64/libglib-2.0.so.0
> #13 0x00007faaa50cc648 in g_main_context_iterate.isra () at /lib64/libglib-2.0.so.0
> #14 0x00007faaa506bb13 in g_main_context_iteration () at /lib64/libglib-2.0.so.0
> #15 0x00007faaa5ddc45f in QEventDispatcherGlib::processEvents(QFlags<QEventLoop::ProcessEventsFlag>) () at /lib64/libQt6Core.so.6
> #16 0x00007faaa5b0c135 in QCoreApplication::processEvents(QFlags<QEventLoop::ProcessEventsFlag>, int) () at /lib64/libQt6Core.so.6
> #17 0x00007faa9b89685b in Squish::Hook::notifyApplicationExit() () at /home/headless/squish/lib/libsquishhook.so
> #18 0x00007faaa0787a89 in write_event_queue () at /home/headless/squish/lib/libsquishqtwrapper.so
> #19 0x00007faaa078b5de in Squish::QtWrapper::Application::exitAUT(int) () at /home/headless/squish/lib/libsquishqtwrapper.so
> #20 0x00007faa9b8891fc in Squish::Hook::kill() () at /home/headless/squish/lib/libsquishhook.so
> #21 0x00007faa9b897166 in Squish::Hook::handleSend(unsigned int, QString const&, Squish::ArgumentList const&) () at /home/headless/squish/lib/libsquishhook.so
> #22 0x00007faa9b94be74 in Squish::AsyncIPC::IncomingSend::finalize(QDataStream&) () at /home/headless/squish/lib/libsquishhook.so
> #23 0x00007faa9b949561 in Squish::AsyncIPC::onMessageArrived(QByteArray) () at /home/headless/squish/lib/libsquishhook.so
> #24 0x00007faa9b965b41 in Squish::AsyncIPC::qt_static_metacall(QObject*, QMetaObject::Call, int, void**) () at /home/headless/squish/lib/libsquishhook.so
> #25 0x00007faaa5b6ddf8 in void doActivate<false>(QObject*, int, void**) () at /lib64/libQt6Core.so.6
> #26 0x00007faa9b9663b2 in Squish::AsyncMessageChannel::messageArrived(QByteArray) () at /home/headless/squish/lib/libsquishhook.so
> #27 0x00007faa9b950f63 in Squish::TCPMessageChannel::processSingleMessage(QByteArray*) () at /home/headless/squish/lib/libsquishhook.so
> #28 0x00007faa9b9525a1 in Squish::TCPMessageChannel::dispatchMessages(bool) () at /home/headless/squish/lib/libsquishhook.so
> #29 0x00007faaa5b6ddf8 in void doActivate<false>(QObject*, int, void**) () at /lib64/libQt6Core.so.6
> #30 0x00007faaa5b7cbc5 in QSocketNotifier::activated(int, QSocketNotifier::QPrivateSignal) () at /lib64/libQt6Core.so.6
> #31 0x00007faaa5b7d3a5 in QSocketNotifier::event(QEvent*) () at /lib64/libQt6Core.so.6
> #32 0x00007faaa6e4a698 in QApplicationPrivate::notify_helper(QObject*, QEvent*) () at /lib64/libQt6Widgets.so.6
> #33 0x00007faaa5b0c018 in QCoreApplication::notifyInternal2(QObject*, QEvent*) () at /lib64/libQt6Core.so.6
> #34 0x00007faaa5ddd47c in socketNotifierSourceDispatch(_GSource*, int (*)(void*), void*) () at /lib64/libQt6Core.so.6
> #35 0x00007faaa506e48c in g_main_context_dispatch () at /lib64/libglib-2.0.so.0
> #36 0x00007faaa50cc648 in g_main_context_iterate.isra () at /lib64/libglib-2.0.so.0
> #37 0x00007faaa506bb13 in g_main_context_iteration () at /lib64/libglib-2.0.so.0
> #38 0x00007faaa5ddc45f in QEventDispatcherGlib::processEvents(QFlags<QEventLoop::ProcessEventsFlag>) () at /lib64/libQt6Core.so.6
> #39 0x00007faaa5b1912b in QEventLoop::exec(QFlags<QEventLoop::ProcessEventsFlag>) () at /lib64/libQt6Core.so.6
> #40 0x00007faaa5b148fd in QCoreApplication::exec() () at /lib64/libQt6Core.so.6
> #41 0x0000000000451631 in main(int, char**) (argc=6, argv=0x7ffef77d65e8) at /drone/src/src/gui/main.cpp:449
>         resource = {<No data fields>}
>         platform = std::unique_ptr<OCC::Platform> = {get() = 0x1b7bb00}
>         app = <incomplete type>
>         options = {show = true, quitInstance = false, logDir = {d = {d = 0x0, ptr = 0x0, size = 0}, static _empty = 0 u'\000'}, logFile = {d = {d = 0x1c9cf00, ptr = 0x1c9cf10 u"-", size = 1}, static _empty = 0 u'\000'}, logFlush = true, logDebug = true, debugMode = false, userEnforcedLanguage = {d = {d = 0x0, ptr = 0x0, size = 0}, static _empty = 0 u'\000'}, fileToOpen = {d = {d = 0x0, ptr = 0x0, size = 0}, static _empty = 0 u'\000'}}
>         singleApplication = {<QObject> = {<No data fields>}, static staticMetaObject = {d = {superdata = {direct = 0x7faaa5fd0460 <QObject::staticMetaObject>}, stringdata = 0x7faaa7aa6240 <(anonymous namespace)::qt_meta_stringdata_KDSingleApplication>, data = 0x7faaa7aa6160 <qt_meta_data_KDSingleApplication>, static_metacall = 0x7faaa7aa1cc0 <KDSingleApplication::qt_static_metacall(QObject*, QMetaObject::Call, int, void**)>, relatedMetaObjects = 0x0, metaTypes = 0x7faaa7aa8b20 <qt_incomplete_metaTypeArray<(anonymous namespace)::qt_meta_stringdata_KDSingleApplication_t, QtPrivate::TypeAndForceComplete<QString, std::integral_constant<bool, true> >, QtPrivate::TypeAndForceComplete<bool, std::integral_constant<bool, true> >, QtPrivate::TypeAndForceComplete<KDSingleApplication, std::integral_constant<bool, true> >, QtPrivate::TypeAndForceComplete<void, std::integral_constant<bool, false> >, QtPrivate::TypeAndForceComplete<QByteArray const&, std::integral_constant<bool, false> >, QtPrivate::TypeAndForceComplete<bool, std::integral_constant<bool, false> >, QtPrivate::TypeAndForceComplete<QByteArray const&, std::integral_constant<bool, false> >, QtPrivate::TypeAndForceComplete<bool, std::integral_constant<bool, false> >, QtPrivate::TypeAndForceComplete<QByteArray const&, std::integral_constant<bool, false> >, QtPrivate::TypeAndForceComplete<int, std::integral_constant<bool, false> > >>, extradata = 0x0}}, d_ptr = std::unique_ptr<KDSingleApplicationPrivate> = {get() = 0x1c9f1e0}}
>         folderManager = std::unique_ptr<OCC::FolderMan> = {get() = 0x1ca2370}
>         ocApp = std::unique_ptr<OCC::Application> = {get() = 0x1c85720}
> 
> <<<<< STACKTRACE END >>>>>
> ```

Probably fixed in https://github.com/owncloud/client/pull/11390

**Comment by saw-jan** [comment]
> Probably fixed in https://github.com/owncloud/client/pull/11390

So let's run tests in CI without cleanup via UI as of now. Cleanup generally works but sometimes fails with AUT crash.

**Review by Salipa-Gurung — APPROVED** [review]

**Pull request merged**